R/posterior.R

```diff
@@ -676,15 +676,15 @@ top_words_calc <- function(n, measure, show_keyword,
      }
   }
   res <- apply(phi, 1, measuref)
-
   if (show_keyword) {
     for (i in 1:ncol(res)) {
+      inds <- which(res[, i] %in% keywords_raw[[i]])
+      res[inds, i] <- paste(res[inds, i], paste0("[", "\U2713" ,"]"))
       for (j in 1:length(keywords_raw)) {
-         inds <- which(res[,i] %in% keywords_raw[[j]])
-         label <- ifelse(i == j,
-                         paste0("[", "\U2713" ,"]"),
-                         paste0("[", as.character(j), "]"))
-         res[inds, i] <- paste(res[inds, i], label)
+        if (i == j) next
```

Comment:
I will try this code tomorrow, but if we go `next` for `i==j` case, can we check if a topic `i` includes top words that are keywords of topic `j`? (keywords that do not overlap).